staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -731,7 +740,7 @@ func testCacherSendBookmarkEvents(t *testing.T, allowWatchBookmarks, expectedBoo
 					ResourceVersion: fmt.Sprintf("%v", resourceVersion+uint64(i)),
 				}})
 			if err != nil {
-				t.Fatalf("failed to add a pod: %v", err)
+				errorCh <- fmt.Errorf("failed to add a pod: %v", err)
```

Comment:
Upon getting a non-nil error, we should return here? (as apposed to keep trying in the loop)

Reply:
Agreed, will add it.
```suggestion
				errorCh <- fmt.Errorf("failed to add a pod: %v", err)
				return
```